### DISCUSSION TITLE
How should I configure “babel-plugin-react-compiler” when using reactRouter in Vite plugin？

### DISCUSSION THREAD

**Discussion by DhtIsCoding** [remix-run/react-router#12598] [Q&A]
![企业微信截图_17346755462940](https://github.com/user-attachments/assets/617579b5-1537-47f4-b542-6b5f304baac9)
I tried to add babel.config to the root directory but it didn't work, what can i do?

**Comment by arashi-dev** [comment]
Did you find any solution?
